drivers/nutdrv_qx.c

```diff
@@ -1891,9 +1891,8 @@ static int	armac_command(const char *cmd, char *buf, size_t buflen)
 		memset(tmpbuf, 0, sizeof(tmpbuf));
 		tmpbuf[0] = 0xa0 + bytes_to_send;
 		memcpy(tmpbuf + 1, cmd + i, bytes_to_send);
-		ret = usb_control_msg(udev,
-			USB_ENDPOINT_OUT + USB_TYPE_CLASS + USB_RECIP_INTERFACE,
-			0x09, 0x200, 0,
+		ret = usb_interrupt_write(udev,
```

Comment:
As mentioned in #2153 usbmon from manufacturer's software sends interrupt instead command. Without this change `usb_interrupt_read` unable to read any data